applications/CompressiblePotentialFlowApplication/custom_elements/embedded_incompressible_potential_flow_element.cpp

```diff
@@ -12,6 +12,7 @@
 #include "embedded_incompressible_potential_flow_element.h"
 #include "compressible_potential_flow_application_variables.h"
 #include "custom_utilities/potential_flow_utilities.h"
+#include "fluid_dynamics_application_variables.h"
```

Comment:
I am using the PENALTY_COEFFICIENT from the FluidApp here. #6318 should be merged first